src/artemis/external_interaction/callbacks/fgs/tests/test_zocalo_handler.py

```diff
@@ -171,21 +170,21 @@ def test_multiple_results_from_zocalo_sorted_by_total_count_returns_centre_and_b
         multi_crystal_result
     )
     found_centre, found_bbox = callbacks.zocalo_handler.wait_for_results(
-        Point3D(0, 0, 0)
+        create_point(0, 0, 0)
     )
     callbacks.zocalo_handler.zocalo_interactor.wait_for_result.assert_called_once_with(
         100
     )
     expected_centre_motor_coords = (
         dummy_params.experiment_params.grid_position_to_motor_position(
-            Point3D(
-                expected_centre_grid_coords.x - 0.5,
-                expected_centre_grid_coords.y - 0.5,
-                expected_centre_grid_coords.z - 0.5,
+            create_point(
+                expected_centre_grid_coords[0] - 0.5,
+                expected_centre_grid_coords[1] - 0.5,
+                expected_centre_grid_coords[2] - 0.5,
             )
         )
     )
-    assert found_centre == expected_centre_motor_coords
+    np.testing.assert_array_equal(found_centre, expected_centre_motor_coords)
 
     expected_bbox_size = list(map(operator.sub, [8, 8, 7], [2, 2, 2]))
```

Comment:
while we're numpyifying stuff, this could be `... = ndarray([8, 8, 7]) - ndarray([2, 2, 2])`, and in the real code where this happens